README.md

```diff
@@ -416,31 +416,15 @@ lines in the `~/.emacs.d/init.el` file:
 
 ### Ubuntu users
 
-You need to create a `.desktop` file that starts Emacs using the Spacemacs logo instead. In summary:
+You need to create a `.desktop` file that starts Emacs using the Spacemacs logo instead
+of default icon.
 
-Create a new file in `~/.local/share/applications/spacemacs.desktop` with the contents
+You can find the default `.desktop` file example in the [emacs repository][desktop_file_example].
```

Comment:
We'd better tell people to put the desktop file in `~/.local/share/applications`